src/routes/donors.ts

```diff
@@ -8,5 +8,7 @@ router.get('/donations', donorController.getDonations);
 router.post('/donations', donorController.postDonations);
 router.delete('/donations/:donation_id', donorController.deleteDonation);
 router.get('/available-pickup', donorController.availPickup);
+router.put('/donations', donorController.modifyDonation);
```

Comment:
Minor note: I think it would be good to have it be at /donations/:donation_id, instead of making a query param. Sorry about not specifying this earlier. Same for /donors/:donor_id.

Reply:
Added changes!